### PR TITLE
issue 3: fix missing, generic, or wrong return types

### PR DESCRIPTION
**Pull request by David-Owen-Walker**

### CODE REVIEW

File: src/components/departure_list/list.js [hunk 1]
```diff
@@ -21,17 +21,19 @@ function renderDeparture(data: Object): * {
 }
 
 
-class DepartureList extends React.Component {
+class DepartureList extends React.Component<{}> {
```
Comment (David-Owen-Walker) [inline src/components/departure_list/list.js:24]: Strictly speaking, this might not belong in this PR

**Review by Jonathan-Harrison — APPROVED** [review]
